src/plugins/intel_npu/src/compiler/CMakeLists.txt

```diff
@@ -21,6 +21,9 @@ target_include_directories(${TARGET_NAME}
         $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
 )
 
+include_directories(${CMAKE_SOURCE_DIR}/src/plugins/intel_npu/src/backend/include)
+include_directories(${CMAKE_SOURCE_DIR}/src/plugins/intel_npu/src/plugin/include)
```

Comment:
It is generally not a good practice to use `include_directories` for CMakeLists.txt. I also am sure if there are really needed here.

Reply:
we remove this two line, and we think that update in `target_link_libraries` will be enough for our case